engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js

```diff
@@ -35,16 +34,8 @@ angular.module('Bastion.content-hosts').controller('ContentHostPackagesApplicabl
             return $scope.getSelectedPackages().join(' ');
         };
 
-        $scope.getKatelloAgentCommand = function () {
-            return $scope.getSelectedPackages().join(',');
-        };
-
         $scope.performDefaultUpdateAction = function () {
```

Comment:
Now that there's only one default action, may want to think about removing the outer function `performDefaultUpdateAction` and just calling `$scope.performViaRemoteExecution` directly instead. (This applies in lots of places, not just this one) Thoughts?

Reply:
I can update this in the html/view easily but trying to change this in the controller causes the tests to fail with
```
HeadlessChrome 115.0.5790 (Linux 0.0.0) Controller: ContentHostPackagesApplicableController Sets a table. FAILED
        TypeError: Cannot read properties of undefined (reading 'getSelected')
            at $scope.getSelectedPackages (/home/vagrant/katello/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js:9:804)
```

Are you fine with me just changing the view to call `performViaRemoteExecution`